docs/setting-up/server/helm.md

```diff
@@ -81,20 +131,11 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 
 ```sh
 helm show values percona/pmm > values.yaml
-#change needed parameters in values.yaml
-helm install pmm -f values.yaml percona/pmm
-```
-
-### PMM admin password
-
-PMM admin password would be set only on the first deployment. That setting is ignored if PMM was already provisioned and just restarted and/or updated.
 
-If PMM admin password is not set explicitly (default), it will be generated.
-
-To get admin password execute:
+#change needed parameters in values.yaml
+yq -i e '.secret.create |= false' values.yaml
```

Comment:
This is not a common package and I don't see mention of what it is anywhere, nor how to get it

Reply:
but that is not a point, just to demonstrate that something needs to be changed. It also to enable testing: https://github.com/percona/pmm-doc/blob/main/.github/workflows/helm-tests.yml